private/buf/bufgen/generator.go

```diff
@@ -169,15 +169,25 @@ func (g *generator) execPlugins(
 	jobs := make([]func(context.Context) error, 0, len(config.PluginConfigs))
```

Comment:
While you are here, can you remove the `appprotoexecGenerator` argument? It is already available on the generator itself and is unused here.